jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -38,15 +40,23 @@ def test_load_spectrum1d(self):
         assert list(data.keys())[0] == self.label
 
     def test_load_spectrum_list_no_labels(self):
+        # now load three more spectra from a SpectrumList, without labels
         self.spec_app.load_spectrum(self.spec_list)
-        assert len(self.spec_app.app.data_collection) == 5
+        assert len(self.spec_app.app.data_collection) == 4
```

Comment:
I don't understand why this test was working before – `self.spec_list` is a list of three spectra. There is one spectrum [already loaded](https://github.com/spacetelescope/jdaviz/pull/1659/files#diff-5ef5cbec11fb8f2fb8f31547e9995778108c86e1f383e9676e2b9ad52830c1bfR28), so there should be 4 in the collection here.

Reply:
This reuses `self.spec_app` modified in other tests. Maybe the number depends on which test ran first? Is it not cleaner to have each test method use their own `specviz_helper`?

Reply:
Well now that I understand the problem the answer here is pretty clear 😅. The point of making this PR was to prevent Specviz from concatenating spectra and creating a new data item by default. That "extra" data item that made this test pass before was that concatenated spectrum. This PR changes the default behavior of loading `SpectrumList`s, so the concatenated spectrum is no longer computed or loaded by default. So the length of the data collection indeed should change from five (one from the `setup_class` method, three from `load_spectrum(spec_list)`, and one from the concatenation of the previous three spectra) to four (skipping the concatenation).

Reply:
Okay, in that case, maybe no need to refactor the tests... yet.